### PR TITLE
fixed react player alt image and fixed padding for caption

### LINKED ISSUE #75: Remove login icon from React player alternate image and make padding consistent
#### Current Behavior
1. There is a login button inside the image displayed for the React player which is misleading. 
image

2. Padding is inconsistent between the React player and the text above and below it
image

#### Desired Behavior
1. Removing the login icon.
2. Adding 3rem as margin-top for caption element similar to the other elements to make it consistent.

#### Implementation
1. Fix for issue 1
image

2. Fix for issue 2
image

#### Mockups

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by vishnus17**
**Notes for Reviewers**

This PR fixes #75 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
nice work @vishnus17 as you are there can you do a small change, you can see in this below image corner of the player is not in a proper curve can you fix this too

Screenshot 2023-02-25 at 3 24 27 AM

**Comment by vishnus17** [comment]
I don't think I can do that. The image fixing I did was with canva. It was just adding a rectangle over the Login button which I noticed when I was signing up for the early access. I'm not really that much of a graphics guy 😅

**Review by Avi-88 — CHANGES_REQUESTED** [review]
Nice work @vishnus17. About what @Yashsharma1911 suggested , you dont need to edit the image itself @vishnus17. Just make it so that the border radius for that component is not in % but a static value like for e.g 10px. You can inspect using dev tools and find the class which applies the said styling.

File: site/src/App.style.js [hunk 1]
```diff
@@ -218,7 +218,7 @@ export const Main = styled.main`
 
   .caption {
     color: ${({ theme }) => theme.text};
-    margin-top: 0.75rem;
+    margin-top: 3rem;
```
Comment (Nikhil-Ladha) [inline site/src/App.style.js:221]: This is logically not needed, this is a caption and should be close to the image and not same as the callout text below it.

**Comment by Nikhil-Ladha** [comment]
> nice work @vishnus17 as you are there can you do a small change, as you can see the corner of the player in the below image is not in a proper curve can you fix this too
> 
> Screenshot 2023-02-25 at 3 24 27 AM

Where do you see that the corner doesn't have the same radius as the image? 🤔 
Seems same to me, or am I _just_ not able to see it

**Comment by Yashsharma1911** [comment]
> > nice work @vishnus17 as you are there can you do a small change, as you can see the corner of the player in the below image is not in a proper curve can you fix this too
> > Screenshot 2023-02-25 at 3 24 27 AM
> 
> Where do you see that the corner doesn't have the same radius as the image? 🤔 Seems same to me, or am I _just_ not able to see it

yes it is actually not properly curved this is because radius is given in % 

### current
Screenshot 2023-02-28 at 3 02 12 PM

### Desired 
Screenshot 2023-02-28 at 3 03 15 PM

**Comment by leecalcote** [comment]
@vishnus17 following up a final time here... is this something that you intend to complete?

**Comment by adithyaakrishna** [comment]
@vishnus17 The fix should be to change the value from 2.5% to 15px, that should fix the case of two backgrounds being visible at the same time as well :)

**Comment by vishnus17** [comment]
Hi guys. Sorry I was having really busy weeks lately.
@leecalcote . Yes I'll complete this.
@adithyaakrishna Thanks for the help.
I'll complete this by this week.

**Comment by leecalcote** [comment]
Sounds great.

**Comment by vishnus17** [comment]
Hi @Yashsharma1911, @Nikhil-Ladha , @leecalcote , @adithyaakrishna . 
I've removed the padding for the caption element and adjusted the border-radius of the react player image.
Please have a final look and see if its good to be merged.
Thanks.

**Review by adithyaakrishna — APPROVED** [review]
LGTM

**Comment by adithyaakrishna** [comment]
@Nikhil-Ladha Could you please merge this PR?

File: site/src/App.style.js [hunk 2]
```diff
@@ -197,7 +197,7 @@ export const Main = styled.main`
     box-shadow: 0px 3px 20px 4px rgba(0, 179, 159, 0.5);
 
     .react-player__preview {
-      border-radius: 2.5%;
+      border-radius: 1.5%;
```
Comment (Nikhil-Ladha) [inline site/src/App.style.js:200]:
```suggestion
      border-radius: '2rem';
```

**Review by Nikhil-Ladha — APPROVED** [review]

File: site/src/App.style.js [hunk 3]
```diff
@@ -197,7 +197,7 @@ export const Main = styled.main`
     box-shadow: 0px 3px 20px 4px rgba(0, 179, 159, 0.5);
 
     .react-player__preview {
-      border-radius: 2.5%;
+      border-radius: '2rem';
```
Comment (Nikhil-Ladha) [inline site/src/App.style.js:200]:
```suggestion
      border-radius: 1.5%;
```

**Pull request merged**